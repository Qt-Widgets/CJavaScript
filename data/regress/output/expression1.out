0
1
1
1
1
5
15
13
-9
16
3
7
