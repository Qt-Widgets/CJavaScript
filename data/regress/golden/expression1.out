0
1
1
1
true
5
15
13
-9
16
3
7
